service/src/main/resources/config/aou/test/SC2023Q3R1/entity/t_device.json

```diff
@@ -4,42 +4,18 @@
   "attributes": [
     { "type": "SIMPLE", "name": "id", "dataType": "INT64" },
     { "type": "SIMPLE", "name": "name", "dataType": "STRING" },
-    { "type": "KEY_AND_DISPLAY", "name": "vocabulary", "dataType": "STRING" },
-    { "type": "KEY_AND_DISPLAY", "name": "standard_concept", "dataType": "STRING" },
+    { "type": "SIMPLE", "name": "vocabulary_id", "dataType": "STRING" },
+    { "type": "SIMPLE", "name": "vocabulary", "dataType": "STRING" },
```

Comment:
Why the change from a single `KEY_AND_DISPLAY` vocabulary attribute to two separate `SIMPLE` attributes?

Reply:
btw, also in AoU, dataset outputs we provide both `concept_id` and `concept_name` for all concepts - gender_concept_id etc.. All these are fixes for the second round, once we get the cohort building to work as exoected in tanagra for AoU dataset.